contracts/SnappAuction.sol

```diff
@@ -116,7 +116,7 @@ contract SnappAuction is SnappBase {
         require(packedOrders.length % 26 == 0, "Each order should be packed in 26 bytes!");
         uint numOrders = packedOrders.length / 26;
         uint32 currentBatchIndex = standingOrders[accountId].currentBatchIndex;
-        require(currentBatchIndex < MAX_CURRENTBATCHINDEX || numOrders == 0, "Last Standing Order has to annulate previous orders");
+        require(currentBatchIndex < MAX_CURRENTBATCHINDEX || numOrders == 0, "Last standing order has to be nullifying");
```

Comment:
I just noticed that this constant `MAX_CURRENTBATCHINDEX` should be `MAX_CURRENT_BATCH_INDEX`. I will look a little closer at this and see if we can make it shorter.

Reply:
Also, I am not sure I understand the meaning of this validation.